config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -138,4 +138,18 @@ endif # SOC_SERIES_NRF53X
 
 endif # CHIP_DFU_OVER_BT_SMP
 
+if MPSL_FEM
+
+config MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL
+	bool
```

Comment:
The current trend is not to define types to throw an error if someone changed the Kconfig name.